include/boost/geometry/algorithms/detail/sections/sectionalize.hpp

```diff
@@ -783,22 +783,24 @@ inline void enlarge_sections(Sections& sections, Strategy const&)
 
     // It makes section a tiny bit too large, which might cause (a small number)
     // of more comparisons
-    for (typename boost::range_iterator<Sections>::type it = boost::begin(sections);
-        it != boost::end(sections);
-        ++it)
+    for (auto& section : sections)
```

Comment:
Can we be certain that `sections` defines `begin()` and `end()` member functions? Is this a standard container? Becasue if this is a type adapted to Boost.Range concept then range-based for loop may not compile for it.

Reply:
I'll check.